Tests/LibWeb/Text/input/abortsignal-abort.html

```diff
@@ -0,0 +1,12 @@
+<script src="include.js"></script>
+<script>
+    test(() => {
+        function testSignal(signal) {
+            println(`Aborted: ${signal.aborted}`);
+            println(`Reason: "${signal.reason}"`);
+        }
+
+        testSignal(AbortSignal.abort());
+        testSignal(AbortSignal.abort("This is a test"));
```

Comment:
could you make this task to abort `fetch()`? or does our fetch implementation not support aborting yet?